### ISSUE TITLE
Switch to control whether or not needed firmware updates are shown

### ISSUE THREAD

**Issue by exx76** [Kane610/aiounifi#339]
I have found a version of Ubiquiti firmware that works for me and is stable and allows all my integrations to work.  As such, I have stopped upgrading the firmware, and won't upgrade it unless some feature that I simply cannot live without requires it (like the addition of a new piece of hardware, for example).

The current integration displays firmware updates that are needed (and presumably allows for the upgrade to be performed?) in the Updates view under settings.  Because of this, I perpetually have an orange circle with a number in it on my Settings tab on the sidebar.

It would be wonderful if there were a checkbox in the integration config that would allow a user to enable/disable displaying of firmware updates needed.

Thank you for all your hard work on this integration, it really is great!

**Comment by exx76** [comment]
Yes, that's what I ended up doing. Curious to see if they get re-enabled on update, or if they will stay disabled. Thanks!

**Comment by Kane610** [comment]
> Yes, that's what I ended up doing. Curious to see if they get re-enabled on update, or if they will stay disabled. Thanks!

Just dismissing an update will still have the entity logic running in the backend, if you instead disable the entity the logic to check for updates for that entity will stop running so it should hopefully work :)

**Comment by Kane610** [comment]
Closing this for now.

**Comment by exx76** [comment]
I meant on updating of the integration, not the unifi device. I don't
update those, remember? :)

**Comment by Kane610** [comment]
Maybe you lost me. :D

My understanding is that you don't want to see notification in HA gui of available updates for UniFi devices.

My proposal was that in the entity registry disable the `update.device_x` to remove any logic that produces those notifications of available updates. This way you already have the requested configuration aspect available.

**Comment by exx76** [comment]
Right. I did disable those entities, and the problem is solved. But I'm
wondering if future updates to the integration will cause those entities to
be re-enabled, thereby showing the devices as needing to be updated again,
and I'd have to go disabled the entities again.

**Comment by Kane610** [comment]
> if future updates to the integration will cause those entities to be re-enabled, thereby showing the devices as needing to be updated again, and I'd have to go disabled the entities again

The integration doesn't change the disabled flag for entities so shouldn't happen. It could happen if you remove the device from your unifi installation and then reinstall it as that would also clean up the entities.
